src/test/moves/spit_up.test.ts

```diff
@@ -34,8 +36,10 @@ describe("Moves - Spit Up", () => {
     game.override.enemyAbility(Abilities.NONE);
     game.override.enemyLevel(2000);
 
-    game.override.moveset([Moves.SPIT_UP, Moves.SPIT_UP, Moves.SPIT_UP, Moves.SPIT_UP]);
+    game.override.moveset([ spitUp.id, spitUp.id, spitUp.id, spitUp.id ]);
```

Comment:
```suggestion
    game.override.moveset(new Array(4).fill(spitUp.id));
```
NIT: Match previous NIT